variables.tf

```diff
@@ -0,0 +1,61 @@
+# ENVIRONMENT
+variable "confluent_environment_name" {
```

Comment:
Can we name this one just environment_name?
it is a module specially designed for confluent anyway.

Reply:
Done.